server/internal/server/vector_stores.go

```diff
@@ -117,19 +120,43 @@ func (s *S) CreateVectorStore(
 		return nil, status.Errorf(codes.Internal, "transaction: %s", err)
 	}
 
-	return toVectorStoreProto(c, cms), nil
+	cs, err := getChunkingStrategy(req.ChunkingStrategy)
+	if err != nil {
+		return nil, err
+	}
+
+	fileCompleted := int64(0)
+	for _, f := range fs {
+		if _, err := s.createVectorStoreFile(ctx, c, f, cs); err != nil {
+			log.Printf("Failed to add file %q to vector store %q: %s", f.Id, c.VectorStoreID, err)
```

Comment:
I think we should return an error to end user. Currently it is completely hidden?